libzecale/tests/circuits/verification_key_hash_gadget_test.cpp

```diff
@@ -101,6 +229,15 @@ void verification_key_hash_gadget_test()
     ASSERT_EQ(nvk_hash_value, hash_value);
 }
 
+TEST(VerificationKeyHashTest, HashTest)
+{
+    using wpp = libff::bw6_761_pp;
+    verification_key_hash_test<
+        wpp,
+        libzecale::groth16_verifier_parameters<wpp>,
```

Comment:
What about renaming the `nverifierT` template param as `nverifierParamsT` to better align with the actual use?

Reply:
`nverifierParamsT` would end up pretty long, so I'm not sure it's worth adding `Params`.  (`nverifierT` is currently used pretty consistently throughout the code.)

Reply:
> (nverifierT is currently used pretty consistently throughout the code.)

Indeed.
I was mostly arguing that what this is instantiated with is the parameters classes and not the "verifier gadgets" etc.
We can stick with that for now, that's not a big deal.